com.unity.multiplayer.mlapi/Runtime/Core/NetworkTickSystem.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using UnityEngine;
+
+namespace MLAPI
+{
+    public class NetworkTickSystem : INetworkUpdateSystem, IDisposable
+    {
+        private const double k_DefaultTickDuration = 0.016;
```

Comment:
hmm, 62.5 ticks per sec. do you mind inlining 1/0.016 math here and also explain what this variable is and why 62.5 tps instead of 20 or 30 or something else for now?

Reply:
The game I shipped in the past used integer number of millis to be able to do all computation on integers. So, that's why 16 milliseconds felt natural. But I don't really care since we're using floats now. Do you insist on making this 1/60f ?

Reply:
you can keep 0.016 if you want but I'm asking for a comment explaining its math + why 62.5fps specifically instead of any other number.